src/serve/options.ts

```diff
@@ -74,15 +76,16 @@ export const processOptions = <TResponse extends Response = Response, TInitialPa
     },
     receiver: receiverEnvironmentVariablesSet
       ? new Receiver({
-          currentSigningKey: environment.QSTASH_CURRENT_SIGNING_KEY!,
-          nextSigningKey: environment.QSTASH_NEXT_SIGNING_KEY!,
-        })
+        currentSigningKey: environment.QSTASH_CURRENT_SIGNING_KEY!,
+        nextSigningKey: environment.QSTASH_NEXT_SIGNING_KEY!,
+      })
       : undefined,
     baseUrl: environment.UPSTASH_WORKFLOW_URL,
     env: environment,
     retries: DEFAULT_RETRIES,
     useJSONContent: false,
     disableTelemetry: false,
+    schema: z.any(),
```

Comment:
we can add schema to non-required fields https://github.com/upstash/workflow-js/blob/github-workflows/src/serve/options.ts#L24 and not pass anything here